File: SchoolRegister/SchoolRegister.Models/Dtos/PersonDto.cs
using SchoolRegister.Models.Enums;

namespace SchoolRegister.Models.Dtos
{
    public class PersonDto
    {
        public int Id { get; set; }

        public string FirstName { get; set; }

        public string MiddleName { get; set; }

        public string LastName { get; set; }

        public string Uic { get; set; }

        public string BirthDate { get; set; }

        public Gender Gender { get; set; }

        public string BirthPlace { get; set; }
    }
}

File: school-register-angular/src/person/dtos/person-dto.ts
export class PersonDto {
  id: number;
  firstName: string;
  middleName: string;
  lastName: string;
  uic: string;
  birthDate: string;
  gender: Gender;
  birthPlace: string;
}

export enum Gender {
  Male = 1,
  Female = 2,
}

export const GenderEnumLocalization = {
  [Gender.Male]: "Мъж",
  [Gender.Female]: "Жена",
};
